clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -448,6 +449,7 @@
     private final long retryBackoffMs;
     private long requestTimeoutMs;
     private boolean closed = false;
+    private final ConsumerInterceptors<K, V> interceptors;
```

Comment:
Nitpick: I think it's a good practice to group all the final fields together (so that one can easily see where the mutable state is).